Hope2 (updated the code associated with the website files, update the readme files for guidance)

AjeshStavir opened the pull request:
Updated the region associated with aws as well as the table name to "shared-list".
Updated quick-shopping-list with Shared List.
Updated email id to hr@stavir.com.
Updated README.me with the configuration made inside AWS (lambda, API Gateway, S3, Cloudfront).

AjeshStavir committed:
updated lambda function with new table name and the exact region

AjeshStavir committed:
Updated quick shopping List with Shared List, Updated the relative path for about, story and privacy policy in the website, updated the email id

skStavir commented on README.md:
```diff
@@ -34,3 +34,47 @@ Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protrac
+ Getshoppinglist for generating an id and for calling with that id while updateshoppinglist for update the list with that given id and CORS_function to handle HTTP requests(get,put,options) and to allow CORS( 
+ Cross Origin Resource Sharing). 
+ Gave necessary IAM permissions in the form of role to the lambda functions so as toIntegrate it with the dynamodb table.
+ Testing the lambda function with appropriate payload (payload not for CORS_function)was done successfully.    
```
Was done successfully does not sound right here. Specify this as an instruction for others to follow

skStavir commented on README.md:
```diff
@@ -34,3 +34,47 @@ Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protrac
+
+So Get and GET {id} was integrated with getshoppinglist lambda function and PUT on to updateshoppinglist function while OPTIONS on to CORS_function through
+‘Manage integration’.
+
```
Add some screenshots here

AjeshStavir replied:
Unable to pin screenshot inside the file.

skStavir commented on README.md:
```diff
@@ -34,3 +34,47 @@ Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protrac
+
+## S3 Bucket configuration
+
+After successful build of the application soon after making changes within the code in angular js format, using the command ‘ng build’, the files generated within the ‘dist’ folder has to be uploaded to the ‘stavir’ S3 bucket and made public with the changes.
```
Specify the complete  ng build comment for production

skStavir commented on README.md:
```diff
@@ -34,3 +34,47 @@ Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protrac
+
+A file named shoppingitems.json was also uploaded to make the suggestion work whenever a user types the first or two letter(s) of the item.
+
+The path to the index was given as ‘stavir.com/tools/sharedlist’.
```
Do not specify was given as. Maybe you can specify as an example

skStavir reviewed (commented):
Please see the individual comments on read me and fix it

AjeshStavir committed:
Update README.md

Readme.md updated with sentence structure changes.
Unable to pin screenshots inside the Readme.md file.

AjeshStavir committed:
Update README.md

Updated Readme.md with appropriate screenshots.
